### DISCUSSION TITLE
redirect to https

### DISCUSSION THREAD

**Discussion by bitcldr** [Dotnet-Boxed/Templates#93] [Q&A] [answered]
Page is redirected to **https**://localhost:5000 when launch _dnx web_.

**Comment by RehanSaeed** [comment, marked as answer]
I guess you are doing this from the command line and not from visual studio. When using the command line, you need to pass the server URL yourself to start the site with the correct scheme and port (The default is always 5000). If you specify this URL after the 'web' command in project.json, then IIS Express stops working :( Maybe RC2 will fix this problem.

**Comment by RehanSaeed** [comment]
Feel free to reopen this issue if you have any more questions.
